app.freelens.Freelens.metainfo.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>app.freelens.Freelens</id>
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+  <name>Freelens</name>
+  <developer_name>Freelens Authors</developer_name>
+  <url type="homepage">https://freelens.app</url>
+  <summary>FOSS UI for Kubernetes</summary>
+  <description>
+    <p>The Kubernetes management tool</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/freelensapp/freelens/8c32ca48bb5ec0befbc66a1c040b404fc4987bf2/.github/screenshot.png</image>
+      <caption>Freelens application</caption>
+    </screenshot>
+  </screenshots>
+  <launchable type="desktop-id">app.freelens.Freelens</launchable>
```

Comment:
this should match the desktop filename

(this is the build failure)

Reply:
Ah crap... Ok, this is my first time with Flatpak package. Thanks for your patience.